src/main/java/domain/LottoTickets.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTickets {
+
+    private static final int INITIAL_NUMBER = 0;
+    private final NumberGenerator numberGenerator;
```

Comment:
```suggestion
    private static final int INITIAL_NUMBER = 0;
    
    private final NumberGenerator numberGenerator;
```

일급 컬렉션에 다른 필드가 있으면 안됩니다!
NumberGenerator도 매개변수로 받아서 사용하는 건 어떨까요?